.storybook/webpack.config.js

```diff
@@ -0,0 +1,44 @@
+// module.exports = {
+//   module: {
+//     rules: [
+//       {
+//         test: /\.css$/,
+//         use: [
+//           { loader: 'style-loader' },
+//           {
+//             loader: 'css-loader',
+//             options: {
+//               importLoaders: 1,
+//               modules: true,
+//               localIdentName: '[name]__[emoji]__[hash:base64:5]',
+//             },
+//           },
+//           {
+//             loader: 'postcss-loader',
+//             options: {
+//               ident: 'postcss',
+//               postcss: {},
+//             },
+//           },
+//         ],
+//       },
+//     ],
+//   },
+// };
+
+module.exports = function ({ config }) {
```

Comment:
Not 100% sure, but I think the base config is https://github.com/storybooks/storybook/blob/2339ce7fc3a8ab3924fe7397f8d7ff3a6897f116/lib/core/src/server/preview/base-webpack.config.js